libecho/tests/tmp_test.cpp

```diff
@@ -35,6 +34,16 @@ TEST(test_send, tmp_test) {
     echo.send(vbuffer);
 }*/
 
+TEST(test_send_receive, tmp_test) {
+    uint8_t buff[10];
+    BitReceiver receiver(14000, 15000, 600, 6);
+    receiver.receiveFirst(buff, 10);
+    for (int i = 0; i < 10; i++) {
+        printf("%02hhx ", buff[i]);
+    }
+}
```

Comment:
This test doesn't really test anything. It also won't be possible to run in CI pipeline. Also, tests should be reproducible.
I'd suggest testing private methods from BitReceiver (by feeding them artificial inputs), preferably by extracting them to some helper class where they are public and can be easily testes, or if that's not possible, you can create test fixture (You can read about text fixture in https://github.com/google/googletest/blob/master/googletest/docs/primer.md ) and set it as friend in BitReceiver to access private methods (I wouldn't recomend going this way).

Reply:
This was just a dummy test to run the syncing and see if it works. I can delete it no problem.
I will test the internal stuff, but not in this PR.

Reply:
Please,delete it